### PR TITLE
Anthem mrx component documentation

### PR DESCRIPTION
**Pull request by tinglis1**
This is the documentation for the anthem_mrx component.
https://github.com/home-assistant/home-assistant/pull/3369

### CODE REVIEW

File: source/_components/media_player.anthem_mrx.markdown [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+description: "Instructions on how to integrate a Anthem MRXx00 Receivers into Home Assistant."
+date: 2016-09-13 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: anthemav.png
+ha_category: Media Player
+featured: false
+ha_release: 0.19
```
Comment (Landrash) [inline source/_components/media_player.anthem_mrx.markdown]: HA release should probaly be 0.29

File: source/_components/media_player.anthem_mrx.markdown [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    host: 192.168.2.200
+    port: 4999
+    mrxzone: 2
+    minvol: -45
+    maxvol: -25
+```
+
+Configuration variables:
+
+- **host** (*Required*): The IP of the IP to Serial device,  eg. `192.168.2.200`
```
Comment (Landrash) [inline source/_components/media_player.anthem_mrx.markdown]: Is this looked to an IP address or could it also hostname?

Comment (tinglis1) [inline source/_components/media_player.anthem_mrx.markdown]: It could be a hostname. I have updated to reflect that.

**Comment by Landrash** [comment]
Parent PR in Home Assistant repo seems to have gone stale. Closing it. You can reopen it when you're ready to finish it.

**Pull request closed without merging**